docker-compose.yml

```diff
@@ -28,11 +28,11 @@ services:
       TOX_WORK_DIR: /tox
       PYTHON_VERSION: py37
 
-  test-admin:
+  admin:
     build:
       context: .
       args:
-        IMAGE: 'python:3.7-slim-stretch'
+        IMAGE: 'python:3.6-stretch'
```

Comment:
turns out celery doesn't work with python 3.7 yet

Reply:
Yeah neither does Django 1.11 :(

Reply:
But this can still be "slim", yeah?